src/Accounts/Accounts/help/Get-AzAccessToken.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Accounts.dll-Help.xml
+Module Name: Az.Accounts
+online version:
+schema: 2.0.0
+---
+
+# Get-AzAccessToken
+
+## SYNOPSIS
+Get raw access token
+
+## SYNTAX
+
+### KnownResourceTypeName
+```
+Get-AzAccessToken -ResourceTypeName <String> [-TenantId <String>] [-DefaultProfile <IAzureContextContainer>]
```

Comment:
What's the difference between ResourceType and RP? I'm confused about this name

Reply:
ResourceTypeName should be same as RP name if possible, but there's no RP name for some services, e.g. Graph, ResourceManager, for those I'll use common used names.